Packet++/header/FtpLayer.h

```diff
@@ -0,0 +1,458 @@
+#ifndef PACKETPP_FTP_LAYER
+#define PACKETPP_FTP_LAYER
+
+#include "LineBasedProtocol.h"
+
+/// @file
+
+/**
+ * \namespace pcpp
+ * \brief The main namespace for the PcapPlusPlus lib
+ */
+namespace pcpp
+{
+
+	/**
+	 * Class for general FTP message
+	 */
+	class FtpLayer : public LineBasedProtocolMessage
+	{
+	protected:
+		FtpLayer(uint8_t *data, size_t dataLen, Layer *prevLayer, Packet *packet) : LineBasedProtocolMessage(data, dataLen, prevLayer, packet) {};
```

Comment:
I think this c'tor can set `m_Protocol = FTP;` instead of setting it in the c'tor of `FtpRequestLayer` and `FtpResponseLayer`